lib/3scale_toolbox/commands/import_command/openapi.rb

```diff
@@ -33,6 +33,8 @@ def self.command
               flag    nil, 'skip-openapi-validation', 'Skip OpenAPI schema validation'
               option  nil, 'oidc-issuer-endpoint', 'OIDC Issuer Endpoint', argument: :required
               option  nil, 'default-credentials-userkey', 'Default credentials policy userkey', argument: :required
+              option  nil, 'override-private-basepath', 'Override the basepath for the public URLs', argument: :required
```

Comment:
I could understand this as that I can override the whole base URL. But the tests show it is only the path component.

But I don't have enough understanding of the topic.

Reply:
All  the information is in the issue #91 

Your are, basically, overriding [basePath](https://swagger.io/specification/v2/) field from the swagger spec and configuring url-rewritting policy accordingly